src/Lecture1.hs

```diff
@@ -77,7 +86,14 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = max - min
+  where
+    list = [x, y, z]
+    max :: Int
+    max = head (reverse (sort list))
```

Comment:
I see that GHC outputs some warnings ⚠️
In this case, it happens because `maximum` and `minimum` are the names of standard functions in Prelude. It still works due to name shadowing. But generally, it's better to avoid name conflicts with standard functions.

It's a bit tough to do this here because both `max` and `maximum` are already taken 😅 But I would use smth like `mx` or `maxVal` (and similar for minimum).